Library/Homebrew/cmd/install-vendor.sh

```diff
@@ -0,0 +1,117 @@
+source "$HOMEBREW_LIBRARY/Homebrew/utils/lock.sh"
+
+if [[ -n "$HOMEBREW_OSX" ]]
+then
+  if [[ "$HOMEBREW_PROCESSOR" = "Intel" ]]
+  then
+    git_URL=""
+    git_SHA=""
+    ruby_URL="https://github.com/xu-cheng/homebrew-portable/releases/download/20160512/portable-ruby-2.0.0-p648.x86_64-darwin9.tar.gz"
```

Comment:
This is for test purpose for time being. We could move my `homebrew-portable` repo to homebrew if needed in the future.

This is built on 64bit on 10.5. I can build 32bit, but I think 64bit is the default for most OS X.